aws/table_aws_wellarchitected_lens_review.go

```diff
@@ -0,0 +1,233 @@
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "lens_name",
+				Description: "The full name of the lens.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LensReview.LensName"),
+			},
+			{
+				Name:        "lens_arn",
+				Description: "The ARN for the lens.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LensReview.LensArn"),
+			},
+			{
+				Name:        "lens_alias",
+				Description: "The alias of the lens.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LensReview.LensAlias"),
+			},
+			{
+				Name:        "workload_id",
+				Description: "The ID assigned to the workload.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "lens_status",
+				Description: "The status of the lens.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LensReview.LensStatus"),
+			},
+			{
+				Name:        "lens_version",
+				Description: "The version of the lens.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LensReview.LensVersion"),
+			},
+			{
+				Name:        "notes",
+				Description: "The notes associated with the workload.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedLensReview,
+				Transform:   transform.FromField("LensReview.Notes"),
+			},
+			{
+				Name:        "updated_at",
+				Description: "The date and time recorded.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("LensReview.UpdatedAt"),
+			},
+			{
+				Name:        "pillar_review_summaries",
+				Description: "A map from risk names to the count of how questions have that rating.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWellArchitectedLensReview,
+				Transform:   transform.FromField("LensReview.PillarReviewSummaries"),
+			},
+			{
+				Name:        "risk_counts",
+				Description: "A map from risk names to the count of how questions have that rating.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("LensReview.RiskCounts"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LensName"),
+			},
+		}),
+	}
+}
+
+type LensReviewInfo struct {
+	WorkloadId *string
+	*types.LensReview
+}
+
+//// LIST FUNCTION
+
+func listWellArchitectedLensReviews(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	workloadId := h.Item.(types.WorkloadSummary).WorkloadId
+
+	// Reduce number of API call if the workload id has been provided in query parameter.
+	equalQuals := d.EqualsQuals
+	if equalQuals["workload_id"] != nil {
+		if equalQuals["workload_id"].GetStringValue() != *workloadId {
```

Comment:
can we use `EqualsQualString` instead of `EqualsQuals`?

Reply:
Updated the code to use `EqualsQualString` function instead.